sale_order_type/views/sale_order_type_view.xml

```diff
@@ -39,8 +39,8 @@
                             />
                             <field name="sequence_id" />
                             <field
-                                name="analytic_account_id"
-                                groups="analytic.group_analytic_accounting"
+                                name="project_id"
+                                groups="project.group_project_user,analytic.group_analytic_accounting"
```

Comment:
Analogous to https://github.com/odoo/odoo/blob/18.0/addons/sale_project/views/sale_order_views.xml#L33, `project.group_project_user` seems enough here. Can you also take `context="{'default_allow_billable': True}"` here as well?